payroll-processor-client/src/environments/environment.ts

```diff
@@ -1,16 +1,8 @@
-// This file can be replaced during build by using the `fileReplacements` array.
-// `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
-// The list of file replacements can be found in `angular.json`.
+import { Environment } from './environment-types';
 
-export const environment = {
-  production: false
+export const environment: Environment = {
```

Comment:
Each `environment` object exported by each `environment.*.ts` file, implements the `Environment` interface to make sure we don't have inconsistent settings between files.

Reply:
Nice! Seeing it with the examples above is a good demonstration.